tee-worker/cli/src/trusted_base_cli/commands/query_storage.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{command_utils::get_worker_api_direct, trusted_cli::TrustedCli, Cli};
+use codec::Decode;
+use itc_rpc_client::direct_client::DirectApi;
+use itp_rpc::{RpcRequest, RpcResponse, RpcReturnValue};
+use itp_types::DirectRequestStatus;
+use itp_utils::FromHexPrefixed;
+use log::{debug, error, warn};
+use my_node_runtime::Hash;
+
+use std::sync::mpsc::channel;
+
+#[derive(Parser)]
+pub struct QueryStorageCommand {
+	/// Storage Key
+	module: String,
+
+	storage: String,
+}
+impl QueryStorageCommand {
+	pub(crate) fn run(&self, cli: &Cli, trusted_args: &TrustedCli) {
+		let direct_api = get_worker_api_direct(cli);
+
+		let key = storage_value_key(self.module.as_str(), self.storage.as_str());
+		let key = format!("0x{}", hex::encode(key));
+
+		let jsonrpc_call: String = RpcRequest::compose_jsonrpc_call(
+			"state_getStorage".to_string(),
+			vec![trusted_args.mrenclave.clone(), key],
+		)
+		.unwrap();
+
+		debug!("setup sender and receiver");
+		let (sender, receiver) = channel();
+		direct_api.watch(jsonrpc_call, sender);
+
+		debug!("waiting for rpc response");
+		match receiver.recv() {
+			Ok(response) => {
+				let response: RpcResponse = serde_json::from_str(&response).unwrap();
+				if let Ok(return_value) = RpcReturnValue::from_hex(&response.result) {
+					warn!("return_value:{:?}", &return_value);
+					match return_value.status {
+						DirectRequestStatus::Ok => {
+							println!("0x{}", hex::encode(return_value.value));
+						},
+						DirectRequestStatus::Error => {
+							warn!("request status is error");
+							if let Ok(value) = String::decode(&mut return_value.value.as_slice()) {
+								warn!("[Error] {}", value);
+							}
+						},
+						DirectRequestStatus::TrustedOperationStatus(status) => {
+							warn!("request status is: {:?}", status);
+							if let Ok(value) = Hash::decode(&mut return_value.value.as_slice()) {
+								warn!("Trusted call {:?} is {:?}", value, status);
+							}
+						},
+					}
+				};
+			},
```

Comment:
I think the response value should be returned as `Result<Vec<u8>>` instead of just printing a `String` format log. The return value can always be a `Vec<u8>`, but decoding to a `String` may fail.